src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -2171,6 +2171,7 @@ public Response getAssignments(@Context ContainerRequestContext crc, @PathParam(
 
     @GET
     @AuthRequired
+    @Deprecated(forRemoval = true, since = "2024-10-17")
     @Path("{id}/privateUrl")
     public Response getPrivateUrlData(@Context ContainerRequestContext crc, @PathParam("id") String idSupplied) {
         return response( req -> {
```

Comment:
Untested but you should be able to replace the whole body with this...

```
return getPreviewUrlData(crc, idSupplied);
```

... and the same pattern should work for the other two methods below. That way we don't have to maintain code in two places.